capture/configs/aes_sca_cw310.yaml

```diff
@@ -1,9 +1,12 @@
-cwfpgahusky:
-  fpga_bitstream: "objs/lowrisc_systems_chip_earlgrey_cw310_0.1.bit"
+target:
+  target_type: cw310
+  fpga_bitstream: "../objs/lowrisc_systems_chip_earlgrey_cw310_0.1.bit"
   force_program_bitstream: False
-  fw_bin: "objs/aes_serial_fpga_cw310.bin"
+  fw_bin: "../objs/aes_serial_fpga_cw310.bin"
+  pll_frequency: 100000000
   baudrate: 115200
   output_len_bytes: 16
+husky:
   pll_frequency: 100000000
   target_clk_mult: 0.1
```

Comment:
If the scope is configured in cycles, those values need to be handed to the scope - hence hey are there I suppose. This is why I didn't like the cfg in cycles for the scope so much ;)

Reply:
As @johannheyszl mentioned, the reason for having these parameters is that they are required for the scope setup.

Reply:
I thought we agreed that this should not be part of the scope code, no?

What I understood was that 1) we specify number of target clock cycles and the target clock frequency, 2) we define the sampling frequency of the scope (either by reading the configured value or by computing it for Husky based on other settings), 3) and then compute what the resulting offsets are which is still not related to the scope.

The scope code should then just get the sampling rate + the window length and offset. Should we set up a quick meeting to avoid any future misunderstandings and confusion?

Reply:
@johannheyszl , @nasahlpa and I just had a quick discussion and agreed that the scope setup code should indeed just get the sampling rate, window length and offset (both in terms of samples). But the user should be able to specify these values in terms of target clock cycles. The function doing the conversion can be a library function called in the test code. This also allows easily manually overriding the values passed to the scope.

A helper function should be created to extract the current sampling rate from the scope and feed that into the conversion function.

We've also said that this functionality can be added in a separate PR to no further delay this one.

Reply:
BTW: with the latest bitstream you have to adjust this from 0.1 to 0.24. Maybe that's why CI is currently failing.

Reply:
Ah thanks, yes this was the issue.